code/iaas/logic/src/main/java/io/cattle/platform/process/externalevent/ExternalEventCreate.java

```diff
@@ -79,6 +95,8 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
             handleVolumeEvent(event, state, process);
         } else if (ExternalEventConstants.KIND_STORAGE_POOL_EVENT.equals(event.getKind())) {
             handleStoragePoolEvent(event, state, process);
+        } else if (ExternalEventConstants.KIND_EXTERNAL_DNS_EVENT.equals(event.getKind())) {
+            handleServiceEvent(event, state, process);
```

Comment:
I know I already LGTMd, but I just realized that my k8s PR has lots of references to `exrternalServiceEvent`. Do you think you could rename this method handleDnsEvent so that we don't have to worry about confusion once both are merged?